README.md

```diff
@@ -3,289 +3,158 @@
 [![Build Status](https://travis-ci.org/Polymer/polymer-cli.svg?branch=master)](https://travis-ci.org/Polymer/polymer-cli)
 [![Build status](https://ci.appveyor.com/api/projects/status/3xc7rkapu39rw9fs/branch/master?svg=true)](https://ci.appveyor.com/project/justinfagnani/polymer-cli/branch/master)
 
-A command-line tool for Polymer projects.
+The command-line tool for Polymer projects and Web Components.
 
-For detailed help, check out the
-[Polymer CLI](https://www.polymer-project.org/1.0/docs/tools/polymer-cli)
-guide.
+## Features
 
-## Overview
+  - **init** - Create a new Polymer project from pre-configured starter templates
+  - **install** - Install bower dependencies as well as [dependency variants](https://www.polymer-project.org/2.0/docs/glossary#dependency-variants) for testing
```

Comment:
IMO, the glossary link doesn't provide enough detail to replace the variants section here.